src/python/datapreprocessor/datapreprocessor/personclient.py

```diff
@@ -0,0 +1,33 @@
+from ipersonclient import IPersonClient
+
+class PersonClient(IPersonClient):
+    def get_all(self):
```

Comment:
What happens if the response is bad? Shouldn't we at least catch it and display a more detailed error, even if we just re-raise whatever exception occurs (which will probably be either a key error or a json decode error). I'd actually suggest you may want to have _get be responsible for doing any such error checking and converting into a dictionary structure so that everything around the requests library is self contained.